config/hubs/2i2c.cluster.yaml

```
@@ -450,3 +450,87 @@ hubs:
                 - swalker
                 - shaolintl
               admin_users: *aup_users
+  - name: utexas-demo
+    domain: utexas-demo.pilot.2i2c.cloud
+    template: basehub
```

```suggestion
    helm_chart: basehub
```

See https://github.com/2i2c-org/infrastructure/pull/955